org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -266,6 +270,67 @@ SsoViewerSupporter:
     principalId: !Ref scienceSupporterGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-viewer-permission-set-arn' ]
 
+SsoSsmSessionInitiator:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-ssm-session-accessor'
+  StackDescription: 'Role to allow users to initiate AWS SSM sessions'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account:
+        - !Ref ScipoolDevAccount
+        - !Ref ScipoolProdAccount
+        - !Ref ScicompAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref AwsSsmSessionInitiatorGroup
+    permissionSetName: 'ssm-session-initiator'
+    sessionDuration: 'PT12H'
+    inlinePolicy: >-
```

Comment:
@ahayden This is for JC users only and SC users can only see the instance IDs of their own provisioned instances so how about we give this role ssm session access to all resources?

Reply:
I don't think we want to do that... the instanceid is not a secret and shouldn't be used for authorization. If you remove the condition to allow start-session only on resources tagged for the resource owner, we are essentially giving root to all instances for all service catalog users.